playwright/types.ts

```diff
@@ -10,3 +10,22 @@ export interface Mock<G> {
     headers?: Record<string, string>;
     status?: number;
 }
+
+export interface LoggedRequest<ReqData = unknown, ResData = unknown> {
+    timestamp: Date;
+    method: string;
+    path: string;
+    headers?: Record<string, string>;
+    data?: ReqData;
+    mocked: boolean;
+    response: {
+        data?: ResData;
+        headers?:  Record<string, string>;
+        status: number;
+    };
+}
```

Comment:
Oh yea, we could. Moxy doesn't export types. We should extend moxy to do this, but should it be in the scope of this PR or a different task?

Reply:
The types I have copied over are actually more useful than the moxy types, because they use generics. I think we should improve and expose these types when we rework moxy.